### PR TITLE
Fuzz: Add progress bar during `creduce` processing

### PR DESCRIPTION
**Pull request by gmlwns2000**
## Wait with a calm!

image

The `fuzz.py -i -r` took really long time, so I think the progress bar is defiantly needed. Progressbar helps users to understand how much the `creduce` is processed.

I tried to use `tqdm`, and I noticed `tqdm` is not installed on the server. So I implement simple progress bar logic in `fuzz.py`. I wanted add to ETA, but it was too time-consuming job to do during working on HW2 :(

## Modified git ignore

Also, I found the `creduce_bug_*` folders are not in `.gitignore`. So I added it on `.gitignore`.

### CODE REVIEW

File: tests/.gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 /csmith-*
 /test*.c
 /reduce-criteria.sh
+/creduce_bug_*
```
Comment (minseongg) [inline tests/.gitignore:5]: Please add a newline

Comment (gmlwns2000) [reply tests/.gitignore]: Fixed!

**Comment by gmlwns2000** [comment]
> Thanks for your work!
> 
> 1. Please rebase and update on the current repo.
> 2. We installed `tqdm` package on the server. Could you please modify implementation by using `tqdm`? (If it can be done in much simpler way)

I rebased the repository to upstream. And moved to `tqdm`. Now it shows ETA since it is the default feature of `tqdm`.

image

**Comment by minseongg** [comment]
Thanks for the work! I think this feature is very helpful.

**Pull request merged**